Sources/LiveViewNative/Registries/BuiltinRegistry.swift

```diff
@@ -16,10 +16,7 @@ protocol BuiltinRegistryProtocol {
     static func decodeModifier(_ type: ModifierType, from decoder: Decoder) throws -> BuiltinModifier
 }
 
-struct BuiltinRegistry: BuiltinRegistryProtocol {
-    
-    static let attributeDecoder = JSONDecoder()
-    
+struct BuiltinRegistry<R: RootRegistry>: BuiltinRegistryProtocol {
```

Comment:
This makes sense, but can you also remove the `R` generic type and the `context` parameter from the `lookup` method, for consistency?